roles/kubernetes-apps/gateway_api/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+gateway_api_enabled: false
+gateway_api_version: v1.1.0
+gateway_api_experimental_channel: false
+gateway_api_install_url: "https://github.com/kubernetes-sigs/gateway-api/releases/download/{{ gateway_api_version }}/{{ 'experimental' if gateway_api_experimental_channel else 'standard' }}-install.yaml"
```

Comment:
HI @tico88612 

It cannot work in the offline environment.
So it may need to copy CRD yamls as an ansible template.

Reply:
Done. PTAL.